docs/running-on-device.md

```diff
@@ -352,10 +352,14 @@ To configure your app to be built using the `Release` scheme, go to **Product**
 
 During the development process, React Native has loaded your JavaScript code dynamically at runtime. For a production build, you want to pre-package the JavaScript bundle and distribute it inside your application. Doing this requires a code change in your code so that it knows to load the static bundle.
 
-In `AppDelegate.m`, change the default `jsCodeLocation` to point to the static bundle that is built in Release.
+In `AppDelegate.m`, change `jsCodeLocation` to point to the static bundle if you're not in debug mode.
 
 ```objc
+#ifdef DEBUG
+   jsCodeLocation = // Your Static Location
```

Comment:
Can this line be filled out? Ideally people can copy/paste this into their project, if we're going to suggest using #ifdef.

Reply:
Fixed by https://github.com/facebook/react-native-website/pull/495/commits/6c6523958f4d95dae57215bdb200cb82e645494f

Can you merge?